src/config.ts

```diff
@@ -110,15 +110,18 @@ export const clubConfig: {
       },
     },
     {
-      name: 'Team2',
+      name: 'Physio Space',
       project: {
-        name: 'Placeholder2',
-        description: '',
+        name: 'Physio Space',
+        description: 'The physio that’s always at your side',
         images: {
-          bannerURI: undefined,
+          bannerURI: 'https://github.com/ubclaunchpad/physiospace-react/blob/master/showcasethumbnail_PhysioSpace.png',
```

Comment:
this has to be a direct link so we can use it as an `src` 📷 

```suggestion
          bannerURI: 'https://raw.githubusercontent.com/ubclaunchpad/physiospace-react/master/showcasethumbnail_PhysioSpace.png',
```